Formula/xrootd.rb

```diff
@@ -27,6 +28,7 @@ class Xrootd < Formula
   depends_on "pkg-config" => :build
   depends_on "python@3.11" => [:build, :test]
   depends_on "davix"
+  depends_on "isa-l"
```

Comment:
This formula has no bottles for ARM, so adding this dependency will break installs for M1 macOS users.

Reply:
isa-l should work fine on ARM, though, so maybe bottles could be added?

Reply:
I doubt it:

https://github.com/Homebrew/homebrew-core/blob/92bdfbe423671d188edf6c6c8a09c27284e587ef/Formula/isa-l.rb#L20-L21

But let's see. #136075

Reply:
I'm having trouble getting CI to cooperate with a bottle build attempt. In the meantime, I tried building `isa-l` from source on my M1 Mac. It fails with:
```
erasure_code/aarch64/gf_2vect_dot_prod_neon.S:33:1: error: unknown directive
.type gf_2vect_dot_prod_neon, %function
^
make[1]: *** [erasure_code/aarch64/gf_2vect_dot_prod_neon.lo] Error 1
erasure_code/aarch64/gf_3vect_dot_prod_neon.S:33:1: error: unknown directive
.type gf_3vect_dot_prod_neon, %function
^
make[1]: *** [erasure_code/aarch64/gf_3vect_dot_prod_neon.lo] Error 1
```

Reply:
Yup, the failure reproduces in CI at #136075.

Reply:
Given info in https://github.com/intel/isa-l/pull/164 (which is not in a release yet - e.g. https://github.com/intel/isa-l/commit/1187583a979dcd98945c8f01905140c9b78d8d11), the current release v2.30.0 isn't compatible with Apple ARM.

And build is still broken as mentioned in follow up: https://github.com/intel/isa-l/pull/226

Reply:
It's ok on not enabling isa-l for now, or just enabling on x86_64, but it would be good to correct the flag for Python.

Reply:
Dropped isa-l changes for now.